tailwind.config.js

```diff
@@ -1,19 +1,17 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   content: [
-    './src/pages/**/*.{js,ts,jsx,tsx,mdx}',
-    './src/components/**/*.{js,ts,jsx,tsx,mdx}',
-    './src/app/**/*.{js,ts,jsx,tsx,mdx}',
-  ],
-  plugins: [
-    require('flowbite/plugin')
+    "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
   ],
+  plugins: [require("flowbite/plugin")],
   theme: {
```

Comment:
Why the sudden change on the ""?

Reply:
Same with the following, is this eslint?

Reply:
yes, i used eslint on file.  Hence the changes.